addons/website_slides/controllers/main.py

```diff
@@ -909,10 +910,20 @@ def slide_preview(self, slide_id):
         return slide.is_preview
 
     @http.route(['/slides/slide/send_share_email'], type='json', auth='user', website=True)
-    def slide_send_share_email(self, slide_id, email, fullscreen=False):
-        slide = request.env['slide.slide'].browse(int(slide_id))
-        result = slide._send_share_email(email, fullscreen)
-        return result
+    def slide_send_share_email(self, emails, res_id, fullscreen=False):
+        for email in emails.split(','):
+            if not single_email_re.match(email.strip()):
+                return False
+        slide = request.env['slide.slide'].browse(int(res_id))
+        return slide._send_share_email(emails, fullscreen)
+
+    @http.route(['/slides/channel/send_share_email'], type='json', auth='user', website=True)
```

Comment:
can you merge this route with the previous one ?

you can even give the model in the route